common/autoware_auto_tf2/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ if(BUILD_TESTING)
   # Unit test
   ament_add_gtest(test_tf2_autoware_auto_msgs test/test_tf2_autoware_auto_msgs.cpp)
   autoware_set_compile_options(test_tf2_autoware_auto_msgs)
+  # workaround to allow deprecated header to build on both galactic and rolling
+  target_compile_options(test_tf2_autoware_auto_msgs PRIVATE
+    -Wno-error
```

Comment:
Please apply the same solution as in https://github.com/autowarefoundation/autoware.universe/pull/720#discussion_r852040761